app/scripts/filters/util.js

```diff
@@ -348,6 +348,30 @@ angular.module('openshiftConsole')
           .replace(/^./, function(str){ return str.toUpperCase(); });
     };
   })
+  // event messages should simple, but human readable:
+  // Pulled Pod <name>
+  // Unhealthy Pod <name>
+  // Deployment created for DeploymentConfig <name>
+  // etc.
+  // This filter handles the "Reason <optional preposition>"
+  // but not the <name> as we <a> link that in templates.
+  .filter('messageForEvent', function($filter) {
```

Comment:
Its definitely better, but does `Pulled -- Pod mariadb-4-cbgkr` still imply it pulled a pod (instead of an image)?  

Ideally the API for events would have an [actor,verb,object,target](http://activitystrea.ms/specs/json/1.0/) model that would give us what we need to construct "Pulled Image for Pod xyz".  

I'll update your suggestion.